trello-backup.py

```diff
@@ -167,7 +168,6 @@ def get_organization_ids(org_names, api_key, token):
             logger.info(" {0} ({1})".format(board["name"], board["id"]))
             boardContents = requests.get(api_url + "boards/" + board["id"], params=board_payload)
             with io.open(output_directory + "/{0}_".format(board["name"].replace("/", "-")) + epoch_time + ".json",
-                         "w", encoding="utf8") as file:
-                args = dict(sort_keys=True, indent=4)
-                data = json.dumps(boardContents.json(), ensure_ascii=False, **args)
-                file.write(data)
+                         "wb") as file:
```

Comment:
Why is this `wb` rather than `w` though? It's still text rather than binary or am I missing something?

Reply:
json.dump requires the data to be unicode rather than a string. Thus requiring to have a file descriptor as `wd`